zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -509,6 +509,19 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
         }
       });
     }
+    else {
+      BootstrapDialog.confirm({
+      title: '',
+      message : 'Nothing is changed.',
+      callback: function(result) {
+        if (result) {
+          $scope.$apply(function () {
+            $scope.showSetting = false;
+            });
+          }
+        }
+      });
+    }
```

Comment:
Just `$scope.showSetting = false;` wouldn't be enough?
Personally I wouldn't expect alert message in this case.
And after clicking cancel, having 'cancel' and 'ok' button choice feels a bit awkward for me.